api/app/routers/hfi_calc.py

```diff
@@ -141,6 +142,15 @@ def save_request_in_database(request: HFIResultRequest, username: str) -> bool:
     return False
 
 
+def fuel_type_model_to_schema(fuel_type_record: FuelTypeModel) -> FuelType:
+    """ Parse a database model record into a schema record. """
+    return FuelType(id=fuel_type_record.id, description=fuel_type_record.description,
+                    abbrev=fuel_type_record.abbrev,
+                    fuel_type_code=fuel_type_record.fuel_type_code,
+                    percentage_conifer=fuel_type_record.percentage_conifer,
+                    percentage_dead_fir=fuel_type_record.percentage_dead_fir)
```

Comment:
wouldn't it be nice if there were a way to just say `FuelType(fuel_type_record)`, since all the attributes are exactly the same?

Reply:
Oh there is with the `*` operator

Reply:
e.g. `FuelType(** fuel_type_record)`

Reply:
That needs a dictionary - and for some silly reason the model doesn't have anything. I'd want to say `FuelType(**fuel_type_record.dict())` - but I don't have that option. I could do `FuelType(**fuel_type_record.__dict__)` but them I'm messing with internals I probablt shouldn't.

Reply:
For some reason there's an easy translation from schema to model, but I can't find one for going the other way :(